app/views/templates/page_template.scala.html

```diff
@@ -52,7 +42,16 @@
                 signOutUrl = Some(s"/lookup-address/destroySession?timeoutUrl=${t.timeoutUrl}")
             ))
         )
+        <!--[if lte IE 8]><link href='@controllers.routes.Assets.at("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
+        <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.at("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
+        <link href='@controllers.routes.Assets.at("accessible-autocomplete-2.0.2.min.css")' media="screen" rel="stylesheet" type="text/css" />
+    }
 
+    @resolvedJourneyConfig = @{
```

Comment:
Does this compile? It is used before it is declared

Reply:
spun it up locally and checked it, also ran the tests, it: tests and the acceptance tests locally and all looked.

Reply:
Excellent, must do some kind of hoisting